python/ray/tune/schedulers/pbt.py

```diff
@@ -500,17 +524,13 @@ def _exploit(self, trial_executor: "trial_executor.TrialExecutor",
         """
         trial_state = self._trial_state[trial]
         new_state = self._trial_state[trial_to_clone]
-        if not new_state.last_checkpoint:
-            logger.info("[pbt]: no checkpoint for trial."
-                        " Skip exploit for Trial {}".format(trial))
-            return
```

Comment:
hmm should we still keep this?

Reply:
I just moved it to a different location (see above)

Reply:
SGTM